src/index.rs

```diff
@@ -430,6 +494,9 @@ impl Index {
 
         //implementations
         self.implementations.extend(other.implementations);
+
+        //Constant expressions
+        self.constant_expressions.import(other.constant_expressions)
```

Comment:
is this legit? are const-IDs unique between two indices?
and how is this connected to this hw-config stuff?